reduce/src/adapter.ts

```diff
@@ -41,7 +41,7 @@ export const execute: Execute = async (request) => {
   const _get = (val: unknown): number => objectPath.get(val, path)
 
   // Filter undesired values
-  inputData = inputData.filter((val) => !!_get(val))
+  inputData = inputData.filter((val) => !isNaN(_get(val)))
```

Comment:
This was an interesting change to make. I understand that using `!!` can eliminate `0` value outliers, and for price feeds I think that makes sense. Changed it to `!isNaN` since it affected taking an average if the array included `0`, and that other non price-feed use cases may require including 0 in the average.